src/main/java/at/ac/tuwien/kr/alpha/grounder/NonGroundRule.java

```diff
@@ -98,6 +98,14 @@ public int getRuleId() {
 		return ruleId;
 	}
 
+	public List<Literal> getBodyLiterals() {
```

Comment:
Why not just `return new ArrayList<>(rule.getBody());`?

Reply:
Done.